modules/oe_paragraphs_media/oe_paragraphs_media.post_update.php

```diff
@@ -80,3 +80,29 @@ function oe_paragraphs_media_post_update_00003(): void {
     $display->save();
   }
 }
+
+/**
+ * Add Highlighted field to Text with Featured media paragraph.
+ */
+function oe_paragraphs_media_post_update_00004() {
+  $storage = new FileStorage(drupal_get_path('module', 'oe_paragraphs_media') . '/config/post_updates/00004_highlighted_field');
```

Comment:
```suggestion
  $storage = new FileStorage(\Drupal::service('extension.list.module')->getPath('oe_paragraphs_media') . '/config/post_updates/00004_highlighted_field');
```
`drupal_get_path` is deprecated.

Reply:
It works in D9.2 but no need to rewrite later. It is already used in other components and ewcms